sys/include/net/nanocoap.h

```diff
@@ -567,6 +567,54 @@ static inline void *coap_get_token(const coap_pkt_t *pkt)
     return coap_hdr_data_ptr(pkt->hdr);
 }
 
+/**
+ * @brief   Get the token from a CoAP over UDP packet from a buffer
+ *
+ * @warning This function will happily read past the buffer on invalid input.
+ *          Avoid using this function. Use @ref coap_parse_udp instead.
```

Comment:
```suggestion
 *          Avoid using this function. Use @ref coap_parse instead.
```

I guess that's the correct function on `master`.

Reply:
Good catch, thx!